examples/text-utilities/src/text-utilities.js

```diff
@@ -95,8 +101,8 @@ export function onAddLineFragments(context) {
   const document = sketch.fromNative(context.document)
 
   // Iterate over each text layer in the selection, calling our addLineFragments function
-  document.selectedLayers.forEach(layer => {
-    if (layer.type === sketch.Types.Text) {
+  document.selectedLayers.layers.forEach(layer => {
```

Comment:
good catch! I think I'll add a `forEach`, `map` and `reduce` method to the selection so that it reads better.